README.md

```diff
@@ -8,6 +8,24 @@ This is a Ruby on Rails app, and should follow [our Rails app conventions](https
 
 You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies. Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
+## Testing the k8s API view locally
+
+* You will need to update the `Trust relationship` for the `release-assumed` role on the AWS `IAM` control panel using your `fulladmin` account on the `integration` environment. The additional trusted entity that you are adding should look like this - 
+
+```
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::1234567890:role/your.name-developer"
+            },
+            "Action": "sts:AssumeRole"
+        }
+```
+
+This step needs to be repeated for the `staging` environment as the app will show the status for both `integration` and `staging` environments.
```

Comment:
How about production? 🤔

Reply:
I've added some more notes to explain why we don't update the production environment.